### PR TITLE
feat(gcore): introduce `read_unchecked` from `read`

### LINKED ISSUE #1859: Provide unchecked ops at the backend
### Problem to Solve

E.g. each time you call gr_read you expect it to be correct and you don't want to write err len

### Possible Solution

Provide this wrappers dependent on error_len ptr value 

### Notes

_No response_

### PR DESCRIPTION
**Pull request by clearloop**
Resolves #1859 

- add `gr_read_unchecked` based on `gr_read`
- export `gr_read_unchecked` to `gcore`

@gear-tech/dev

### CODE REVIEW

File: core-backend/sandbox/src/funcs.rs [hunk 1]
```diff
@@ -338,6 +338,14 @@ where
         })
     }
 
+    pub fn read_unchecked(ctx: &mut Runtime<E>, args: &[Value]) -> SyscallOutput {
```
Comment (breathx) [inline core-backend/sandbox/src/funcs.rs:341]: we don't need separate syscall for this
use already existing one by playing around value of "resulting" ptr arg: if u32::MAX than mean as infallible

**Review by breathx — CHANGES_REQUESTED** [review]

**Comment by shamilsan** [comment]
I don't see `gr_read_unchecked` in this PR. Maybe it is worth changing PR's title and description?

**Review by SabaunT — APPROVED** [review]
lgtm, but would wait for @breathx review

**Review by breathx — CHANGES_REQUESTED** [review]
It's okay with gcore, but such fn required in gstd also
Going further, where is following changes for backend to ignore errors if error_len ptr is u32::MAX?

**Review by breathx — CHANGES_REQUESTED** [review]

**Review by breathx — CHANGES_REQUESTED** [review]

**Comment by breathx** [comment]
#2083 merged, to be rebased and refactored as discussed personally

**Pull request closed without merging**